shapes/Rectangle.cpp

```diff
@@ -4,6 +4,15 @@
 Rectangle::Rectangle(double x, double y, Color c) : x_(x), y_(y),  Shape(c)
 {}
 
+Rectangle::Rectangle() : x_(1), y_(1)
+{}
+
+
+Rectangle::Rectangle(Color c) : Rectangle()
+{
+    color_ = c;
```

Comment:
you don't need to set it here. Use the constructor `Rectangle(double x, double y, Color c)`. Body will be empty.